src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length - 1; i++) {
+    if (robotValues.includes(robotValues[i], i + 1)) {
+      return null;
+    }
+  }
```

Comment:
you don't need `robotValues` array for this task
you can check if key exist in object with `hasOwnProperty` method

Reply:
but i need to check "If any of the object values are repeated, return 'null'."
For this i use `robotValues ` and loop